store/src/java/com/zimbra/cs/account/ldap/LdapProvisioning.java

```diff
@@ -2476,7 +2478,9 @@ private void addAliasInternal(NamedEntry entry, String alias) throws ServiceExce
             HashMap<String, String> attrs = new HashMap<String, String>();
             attrs.put("+" + Provisioning.A_zimbraMailAlias, alias);
             attrs.put("+" + Provisioning.A_mail, alias);
-
+            if (isHiddenAlias) {
+                attrs.put("+" + Provisioning.A_zimbraAliasListToHide, alias);
```

Comment:
What is the significance of this "+" here?

Reply:
This is the way of telling Ldap to add the alias to the A_zimbraAliasListToHide list. 
You can see the same code in the above lines.